app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
+    client = __elasticsearch__.client
+    settings index: {
+               analysis: {
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.